src/main/java/hudson/plugins/ec2/EC2Cloud.java

```diff
@@ -807,7 +816,7 @@ public Collection<PlannedNode> provision(final Label label, int excessWorkload)
             }
         }
         LOGGER.log(Level.INFO, "We have now {0} computers, waiting for {1} more",
-            new Object[]{jenkinsInstance.getComputers().length, plannedNodes.size()});
+            new Object[]{jenkinsInstance.getComputers().length, plannedNodes.size() - jenkinsInstance.getComputers().length});
```

Comment:
This does not sound right. `plannedNodes` is supposed to be the list of new nodes about to be added, so why would you substract a generally far larger number of agents, including ones that have nothing to do with this plugin at all?

Reply:
because i see log messages that make no sense, such as 1 have 1 computer, waiting for 1 more` when all that was requested was 1 computer.  I am fine to revert this change but i think the original code is also not correct.

Reply:
> I have 1 computer, waiting for 1 more

Perhaps because the one initial computer was `MasterComputer`? Are you running in a system that sets zero built-in executors?